index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Minesweep</title>
+    <link rel="stylesheet" href="style.css" />
+    <script type="module" src="script.js"></script>
+</head>
+
+<body>
+    <header>
+        <h1>Minesweep</h1>
+    </header>
+    <div></div>
+    <main>
+        <article>
+            <div class="gameoptions">
+                <button class="newGame" title="New Game"><img src="restartIcon.svg" width="30" height="30" /></button>
+                <button class="flagButton" title="turn flagging on"><img src="flagIconWhite.svg" width="30" height="30" /></button>
```

Comment:
correct me if I'm wrong, but it's usually best to avoid inline styling.
```suggestion
                <button class="newGame" title="New Game"><img src="restartIcon.svg" class="icon-btn"/></button>
                <button class="flagButton" title="turn flagging on"><img src="flagIconWhite.svg" class="icon-btn" /></button>
```
with something like this in your css
```
.icon-btn {
  width: 30px;
  height: 30px;
}
```

Reply:
why did i do that?

Reply:
It's best to avoid inline styling besides certain situations (when you want it to override a different style)
Changed it